src/main/java/mate/jdbc/dao/DataProcessingException.java

```diff
@@ -0,0 +1,20 @@
+package mate.jdbc.dao;
+
+import java.sql.SQLException;
+import java.util.NoSuchElementException;
+
+public class DataProcessingException extends RuntimeException {
+    private String message;
+    private Throwable ex;
+
+    public DataProcessingException(String message, NoSuchElementException ex) {
```

Comment:
```suggestion
    public DataProcessingException(String message, Throwable ex) {
```
This way you will be able to pass any child as an input